docs/map.xsd

```diff
@@ -59,13 +124,31 @@
   </xs:restriction>
 </xs:simpleType>
 
-<xs:simpleType name="hexColourT">
+  <xs:simpleType name="cornerterrainT">
+    <xs:restriction base="xs:string">
+      <xs:pattern value="[0-9]{1,3},[0-9]{1,3},[0-9]{1,3},[0-9]{1,3}"/>
+    </xs:restriction>
+  </xs:simpleType>
+
+<xs:simpleType name="hexColorT">
   <xs:restriction base="xs:string">
-    <xs:pattern value="[a-fA-F0-9]{6}"/>
+    <xs:pattern value="([a-fA-F0-9][a-fA-F0-9]){3}"/>
```

Comment:
Please do allow an (optional) `#` at the start here. I'd like to always have it there, but for compatibility reasons Tiled doesn't write the `#` on the `trans` attribute.